dev-tools.sh

```diff
@@ -33,6 +33,6 @@
 # stable version.
 #
 # Uncomment and set the Dev-Tools's commit value and commit this change.
-# export GH_COMMIT=COMMIT_SHA
+export GH_COMMIT=9417e2753e852c8241c817e4ee40133014bb0659
```

Comment:
Assuming this won't be committed.

Reply:
Yes, I am reverting back once it is approved.